fs/rpmsgfs/rpmsgfs_client.c

```diff
@@ -727,11 +727,13 @@ int rpmsgfs_client_rename(FAR void *handle, FAR const char *oldpath,
   size_t len;
   size_t oldlen;
   size_t newlen;
+  size_t alignlen;
   uint32_t space;
 
-  oldlen = strlen(oldpath) + 1;
-  newlen = strlen(newpath) + 1;
-  len    = sizeof(*msg) + oldlen + newlen;
+  oldlen   = strlen(oldpath) + 1;
+  alignlen = (oldlen + 0x7) & ~0x7;
```

Comment:
How about creating a macro like https://github.com/apache/incubator-nuttx/blob/f623ac0f13ab893ef0c209325d2ac1ff85f405ee/arch/risc-v/src/esp32c3/esp32c3_dma.c#L62?

Reply:
`ALIGN_UP` is not a standard macro as well as `ARRAY_SIZE`, so usually it is defined in each place individually. If the align up logic is used more than once in the file maybe it is reasonable to add a macro

Reply:
I will let author to reply on this before merging

Reply:
Just a suggestion for improving the readability.

Reply:
Let's keep it the same as rpmsgfs_server.c:rpmsgfs_rename_handler?